src/internal.c

```diff
@@ -37553,7 +37593,7 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
     {
         byte            b;
         ProtocolVersion pv;
-#ifdef WOLFSSL_SMALL_STACK
+#if defined(WOLFSSL_SMALL_STACK) || defined(OPENSSL_EXTRA)
         Suites*         clSuites = NULL;
 #else
         Suites          clSuites[1];
```

Comment:
This is getting too confusing for no reason. Let's change it to `Suites* clSuites = NULL` in all cases. I think it would be a good idea to just get rid of the local `clSuites` var and stick with `ssl->clSuites` in all cases. For optimization we can consider free'ing it immediately in this func without `OPENSSL_EXTRA`.